web/homepage.html

```diff
@@ -14,12 +14,13 @@
  <div id="wrapper">
  
   <h1>Open with LBRY</h1>
-  <div>
-    <input type="text" id="name" placeholder="Search" />
+  
+  <div id="claim">
+    <span>lbry://</span><input type="text" id="name" placeholder="Enter LBRY Claim / Channel here" maxlength="100" />
```

Comment:
i changed the placeholder too in my last commit so you should rebase or anyway merge my changes in your branch to avoid the conflict